x-pack/plugins/uptime/public/components/queries/ping_list/ping_list.tsx

```diff
@@ -197,6 +197,8 @@ export class Pings extends React.Component<PingListProps, PingListState> {
                     <EuiFormRow label="Max Search Size">
                       <EuiFieldNumber
                         defaultValue={this.state.maxSearchSize.toString()}
+                        min={0}
```

Comment:
This LGTM - do you think it'd be better to reference these values through constants, or keep the bare `number` values in the props like you have them here? It doesn't matter very much either way, and we have a comment explaining the choice for the top of the range.

Reply:
I don't think min needs a constant, but I think it's debate-able for the max. Maybe let's see if we need this anywhere else, and wait till then for the refactor?